src/Pages/Logout.js

```diff
@@ -1,13 +1,33 @@
+import {useEffect} from 'react';
 import {useDispatch, useSelector} from 'react-redux';
+import {useNavigate} from 'react-router-dom';
+import {toast} from 'react-toastify';
 import {removeUser} from '../redux/redux-reducers/userstore';
 
 const Logout = () => {
+  const navigate = useNavigate();
   const loggedIn = useSelector((state) => state.userStore.loggedIn);
   const dispatch = useDispatch();
-  if (loggedIn) {
-    dispatch(removeUser());
-    window.location.reload();
-  } else return <h1 style={{color: 'white', fontFamily: 'monospace'}}>You have been logged out, thank you for using our movieApp</h1>;
+
+  dispatch(removeUser());
```

Comment:
This should be a part of either useEffect or some callback

Reply:
I was originally thinking to use logout just as a button function that deletes the user from localStorage and userStore not as a route. What do you think about that idea?

Reply:
That is the way better idea. Great thinking :+1: